shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -599,7 +602,10 @@ void onStop() {
     // screen when unlocked. We can work around this by changing the visibility of FlutterView in
     // onStart and onStop.
     // See https://github.com/flutter/flutter/issues/93276
-    flutterView.setVisibility(View.GONE);
+    if (flutterView.getVisibility() != View.GONE) {
```

Comment:
```suggestion
```
I wonder if this judgment can be removed, or `previousVisibility` needs to be assigned again in else.
So let's say this is the case：
* First time the flutterView is `View.VISIBLE`. When `onStop` is executed, `previousVisibility` will also become `View.VISIBLE`.
* Second time I change the flutterView to `View.GONE`. When `onStop` is executed, the `previousVisibility` will not be changed. 

If `onStart` is executed at this time,` previousVisibility` remains as before, causes the flutterView to be `View.VISIBLE`（Even though I've changed to `View.GONE`）

Reply:
Good catch! I'll remove it.

Reply:
Done